__tests__/redux/actions/bookmark.test.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unreachable */
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable no-undef */
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import { getBoomarks, bookmark } from '../../../src/redux/actions/article.actions';
+import { bookmarkk } from '../../../__mocks__/data';
+import getArticleReducer from '../../../src/redux/reducers/articles.reducer';
+import { GET_BOOKMARK, BOOKMARK } from '../../../src/redux/actions/types/article.type';
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+const store = mockStore({});
+
+describe('SHOULD TEST BOOKMARKS', () => {
```

Comment:
@EmyRukundo the describe is not supposed to have that "SHOULD"

Reply:
Thanks! I will correct it